README.md

```diff
@@ -1,8 +1,40 @@
-### Code for Paper: “Low-Resource” Text Classification: A Parameter-Free Classification Method with Compressors
+# Code for Paper: “Low-Resource” Text Classification: A Parameter-Free Classification Method with Compressors
 
 This paper is accepted to Findings of [ACL2023](https://aclanthology.org/2023.findings-acl.426/).
 
-### Require
+## Getting Started
+
+This codebase is [available on pypi.org via](https://pypi.org/project/npc-gzip) 
+
+
+```bash
+
+pip install npc-gzip
+
+```
+
+## Usage
+
+See the [examples](./examples/imdb.py) directory for example usage.
+
+
+## Testing
+
+This package utilizes `poetry` to maintain its dependencies and `pytest` to execute tests. To get started running the tests:
+
+```bash
+
+poetry shell
+poetry install
+poetry run pytest --doctest-modules --ignore original_codebase/ --ignore examples/
```

Comment:
`poetry run` is redundant here, because we're in a poetry shell.

Reply:
Good catch, I updated the github actions pipelines as well.

Reply:
A script (including CI) can't give `poetry shell` followed by a command to run in that shell, because the next command actually runs in the original shell, after the shell created by `poetry shell` exits. That's the cause of the failures you saw on CI. In contrast, for humans, `poetry shell` followed by other commands intuitively means to run those other commands in the shell created by `poetry shell`, so this kinds of instructions work in documentation.

Reply:
Yeah today I learned. Fixed the pipelines